services/simplified_crontab

```diff
@@ -21,9 +21,26 @@ HOME=/var/www/circulation
 # the collections.
 #
 */20 * * * * root core/bin/run bibliographic_coverage >> /var/log/cron.log 2>&1
-*/10 * * * * root core/bin/run metadata_wrangler_coverage >> /var/log/cron.log 2>&1
 30 3 * * * root core/bin/run update_nyt_bestseller_lists >> /var/log/cron.log 2>&1
 
+# Add newly discovered identifiers to our Metadata Wrangler collection.
+*/10 * * * * root core/bin/run metadata_wrangler_collection_sync >> /var/log/cron.log 2>&1
+
+# Remove newly removed identifiers from our Metadata Wrangler collection
+* */22 * * * root core/bin/run metadata_wrangler_collection_reaper >> /var/log/cron.log 2>&1
+
+# Check whether the Metadata Wrangler has ascertained any new
+# information about books in our collection.
+*/59 * * * * root core/bin/run metadata_wrangler_collection_updates >> /var/log/cron.log 2>&1
```

Comment:
Similar to above, are you trying to say every 59th minute (as a means to offset on different servers) or on the 59th minute of each hour?

Reply:
Every 59th minute.